lib/hls/hls_parser.js

```diff
@@ -3219,12 +3225,20 @@ shaka.hls.HlsParser = class {
     let previousReference = null;
     let lastDiscontinuityStartTime = firstStartTime;
 
+    const bitrates = [];
+
     for (let i = 0; i < hlsSegments.length; i++) {
       const item = hlsSegments[i];
       const startTime =
           (i == 0) ? firstStartTime : previousReference.endTime;
       position = mediaSequenceNumber + skippedSegments + i;
 
+      const bitrate = shaka.hls.Utils.getFirstTagWithNameAsNumber(
```

Comment:
This needs special handling, like EXT-X-MAP.  It applies to every segment between it and the next EXT-X-BITRATE.  See EXT-X-MAP in lib/hls/manifest_text_parser.js, and maybe generalize that so we have a list of tags that apply in that way.

Reply:
Done!